app/scripts/modules/titus/src/serverGroup/configure/wizard/TitusCloneServerGroupModal.tsx

```diff
@@ -0,0 +1,241 @@
+import * as React from 'react';
+import { get } from 'lodash';
+import { FormikErrors, FormikValues } from 'formik';
+
+import {
+  Application,
+  DeployInitializer,
+  FirewallLabels,
+  IModalComponentProps,
+  IStage,
+  ReactInjector,
+  ReactModal,
+  TaskMonitor,
+  WizardModal,
+  noop,
+  AccountTag,
+} from '@spinnaker/core';
+
+import { ServerGroupCapacity, ServerGroupLoadBalancers, ServerGroupSecurityGroups } from '@spinnaker/amazon';
+
+import { ITitusServerGroupCommand } from '../serverGroupConfiguration.service';
+import { TitusReactInjector } from '../../../reactShims';
+
+import { ServerGroupBasicSettings, ServerGroupResources, ServerGroupParameters } from './pages';
+
+export interface ITitusCloneServerGroupModalProps extends IModalComponentProps {
+  title: string;
+  application: Application;
+  command: ITitusServerGroupCommand;
+}
+
+export interface ITitusCloneServerGroupModalState {
+  firewallsLabel: string;
+  loadBalancerNote: React.ReactElement<any>;
+  loaded: boolean;
+  requiresTemplateSelection: boolean;
+  securityGroupNote: React.ReactElement<any>;
+  taskMonitor: TaskMonitor;
+}
+
+type CloneServerGroupModal = new () => WizardModal<ITitusServerGroupCommand>;
```

Comment:
use TSX generic style as seen in https://github.com/spinnaker/deck/pull/5756/commits/0440735d6fc0c132a74b579576fe8f64101c9cfa